website/docs/r/cognito_identity_provider.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "aws"
+page_title: "AWS: aws_cognito_identity_provider"
+side_bar_current: "docs-aws-resource-cognito-identity-provider"
+description: |-
+  Provides a Cognito User Identity Provider resource.
+---
+
+# aws_cognito_identity_provider
+
+Provides a Cognito User Identity Provider resource.
+
+## Example Usage
+
+### Basic configuration
+
+```hcl
+resource "aws_cognito_user_pool" "example" {
+  name 						= "example-pool"
```

Comment:
Minor nitpick: the formatting is slightly off in the example configuration here and below with email